### PR TITLE
GMMS-6209: Build a RC version from develop

### PR DESCRIPTION
**Pull request by mobichel**

### CODE REVIEW

File: GCMessengerSDKSample/app/build.gradle [hunk 1]
```diff
@@ -51,9 +51,9 @@ android {
 dependencies {
 
     //region Genesys
-    implementation("com.genesys.cloud:core:1.10.2.rc1")
-    implementation("com.genesys.cloud:chatintegration:1.10.2.rc1")
-    implementation("com.genesys.cloud:ui:1.10.2.rc1")
+    implementation("com.genesys.cloud:core:1.10.2.rc2")
```
Comment (SzaszaGenesys) [inline GCMessengerSDKSample/app/build.gradle:54]: Please increase the versionCode too.

**Review by SzaszaGenesys — CHANGES_REQUESTED** [review]

File: GCMessengerSDKSample/app/build.gradle (hunk 1 above)
Comment (SzaszaGenesys) [reply GCMessengerSDKSample/app/build.gradle:54]: This should be 56. 55 is my current release. We do need a solution for this :D

**Review by SzaszaGenesys — APPROVED** [review]

File: GCMessengerSDKSample/app/build.gradle (hunk 1 above)
Comment (mobichel) [reply GCMessengerSDKSample/app/build.gradle:54]: @SzaszaGenesys , should we really increase `versionCode` when increase `rc` version? 

If so I can propose the following way of `versionCode` construction:

```
    def projectVersion = "1.10.2.rc"
    def parts = projectVersion.split('\\.')
    def major = parts[0].toInteger()
    def minor = parts[1].toInteger()
    def patch = parts[2].toInteger()
    
      def versionCode = (major * 1000000) + (minor * 10000) + (patch * 100)
    
    if (parts.length > 3) {
        def preRelease = parts[3].replaceAll('rc', '').toInteger()
        versionCode += preRelease
    }
```

"1.10.2.rc2" becomes 1100202
"1.10.2" becomes 1100200
"2.0.0" becomes 2000000
"0.9.5.rc1" becomes 9501

Comment (SzaszaGenesys) [reply GCMessengerSDKSample/app/build.gradle:54]: @mobichel Sounds good. I guess you can create a ticket for it.

**Pull request merged**